package.json

```diff
@@ -13,15 +13,19 @@
   "devDependencies": {
     "@types/dotenv": "^8.2.0",
     "@types/express": "^4.17.21",
+    "@types/express-fileupload": "^1.4.4",
```

Comment:
You're not using these anymore. remove from dependencies keep our dependencies clean